development/extensions/index.md

```diff
@@ -49,26 +120,35 @@ For an Extension to be properly managed and shareable, the Dockerfile it's creat
    - For reference, existing permissions can be retrieved from a running Docker Container by running the command `docker inspect <container_name>`
       - Do not copy the whole output to use in your Extension - only the minimal requirements for the Extension to work
    - Common uses of the `permissions` label include:
-      - Mapping/mounting a folder from the host device into the running Container
+      - Mapping/mounting a folder from the host device into the running Container, so the extension can **store persistent files**
+         - Useful for storing logs, configuration files, and recorded data
          ```dockerfile
-         "HostConfig":\
-           {\
-             {\
-               {\
-                 "Binds":["/var/lib/zerotier-one:/var/lib/zerotier-one"]\
-               }\
-             }\
-           }
+         "HostConfig": {\
+           "Binds":[\
+             "/usr/blueos/extensions/data-logger:/app",\
```

Comment:
```suggestion
             "/usr/blueos/userdata/extensions/data-logger:/app",\
```

Reply:
This suggestion doesn't match [the current implementation](https://github.com/bluerobotics/BlueOS/pull/2010/files), which follows [the decision we made](https://github.com/bluerobotics/BlueOS/issues/1640#:~:text=We%20ended%20up%20deciding%20the%20directory%20structure%20of%20the%20binding%20should%20be%20/usr/blueos/extensions/%3Cextension_name%3E/), and is what I've documented.

If you think the `extensions` folder should instead be inside `userdata` then you'll have to raise that as a discussion point, but given extensions can have their own logs and the like then I don't think it really fits.